tests/02a-checkrun_with_self_setup.sh

```diff
@@ -5,12 +5,10 @@ source ./tests/testing_lib.sh
 
 # Check if cwd is correctly switched when using --self-setup option
 check_message_in_output "ansible-deployer run -t task_exec_bin_true -s testing -i testInfra -d --self-setup /etc/alt-workdir" "\[DEBUG\]: Successfully created workdir: /tmp/"
-check_message_in_output "ansible-deployer run -t task_exec_bin_true -s testing -i testInfra -d --self-setup /etc/alt-workdir" "\[INFO\]: Setup completed in /etc/alt-workdir"
```

Comment:
maybe we should check if it's not in output? We had a bug so we need a regression test against the fix